metrics.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+import numpy as np
+import torch
+from scipy import linalg
+from torch.nn.functional import adaptive_avg_pool2d
+import torchvision
+from pytorch_fid.inception import InceptionV3
+from torchmetrics.image.inception import InceptionScore
+
+
+class FID():
```

Comment:
It would be better if you put this file not in the root directory. We already have an `eval` module which would be a natural fit for this.